styleguide/components/11-base/forms/_forms.scss

```diff
@@ -120,3 +119,248 @@ $field-icon-padding: 2.5rem;
+    //
+    + label {
+      @include theme('color', 'color-tertiary', 'label-color');
+
+      // Hide checkmark by default.
+      &::after {
+        content: none;
+      }
+    }
+
+    //
+    // Checkbox specific success state.
+    // Checkbox specific error state.
+    //
+    [class*="cs--"] & {
+      &.success + label::before {
+        border-color: color('green');
+      }
+
+      &.error + label::before {
+        border-color: color('red');
+      }
+    }
+
+    //
+    // @TODO: Update when a focus style has been designed.
+    //
+    &:focus + label {
+      outline: 1px dotted;
+    }
+
+    //
+    // Disabled state.
+    //
+    &:disabled {
+      + label {
+        font-weight: normal !important; // sass-lint:disable-line no-important
+        cursor: not-allowed;
+
+        &::before {
+          border-color: color('dark-gray', -4) !important; // sass-lint:disable-line no-important
+          background: color('dark-gray', -4) !important; // sass-lint:disable-line no-important
+        }
+
+        &::after {
+          color: color('dark-gray') !important; // sass-lint:disable-line no-important
+        }
+      }
+    }
+
+    //
+    // Checkbox specific styling.
+    //
+    @if $type == 'checkbox' {
+      // Checked state.
+      &:checked {
+        // Show checkmark on checked.
+        + label {
+          @include icon('checkmark', 'after');
+
+          font-weight: 700;
+
+          &::after {
+            font-size: 1.2rem;
+          }
+        }
+
+        + label::before {
+          @include theme('background-color', 'color-primary--lighten-5', 'checkbox-background-color-checked');
+          @include theme('border-color', 'color-primary', 'checkbox-border-color-checked');
+        }
+
+        + label::after {
+          @include theme('color', 'color-primary', 'checkbox-checkmark-color');
+        }
+      }
+    }
+
+    //
+    // Radio specific styling.
+    //
+    @if $type == 'radio' {
+      // Checked state.
+      &:checked {
+        // Show checkmark on checked.
+        + label {
+          font-weight: 700;
+        }
+
+        + label::before {
+          @include theme('background-color', 'color-primary--lighten-5', 'radio-background-color-checked');
+          @include theme('border-color', 'color-primary', 'radio-border-color-checked');
+        }
+
+        + label::after {
+          @include theme('color', 'color-primary', 'radio-checkmark-color');
+          content: '';
+        }
+      }
+    }
+  }
+}
+
+.form-item {
+  margin-bottom: 1rem;
+
+  .form-label {
+    > label {
+      @at-root .no-optional .label-optional {
+        display: none;
+      }
+    }
+  }
+
+  .form-columns {
+    display: flex;
+    flex-wrap: wrap;
+
+    .form-item-column {
+      @include tablet {
+        width: calc(50% - #{$gutter-width});
+        margin-right: $gutter-width;
+      }
+
+      width: 100%;
+
+      &:first-child {
+        @include tablet {
+          order: 2;
```

Comment:
Be very careful when changing logical order.
https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html

Why was this done?

Reply:
This is done because the order on mobile/tablet of the information boxes changes.